docs/explainer.md

```diff
@@ -1,137 +1,1058 @@
-For MiniApp, there are some pre-standard implementations and wildly used. But these implementations may follow different vendor-specific development guidelines for developing MiniApp UI components. And some of them are defining their own markup languages (WXML, SWAN, AXML), specific event handling, data-binding and rending methods. For example:
 
-- Wechat WXML:
+Although MiniApps Components share the same concept as Web Components, their capabilities and development features differ. The main differences are motivated by the architecture of the existing implementations along the MiniApp ecosystem following the MVVM pattern, based on components developed through domain-specific markup languages and without access to the standard DOM. 
+
+Even though the MiniApp Packaging specification recommends that MiniApp agents support standard Web Components, MiniApp vendors might opt to implement the internal components using the traditional MVVM architecture and, subsequently, manage them through non-standard mechanisms like virtual DOM (VDOM) or content interpolation.
+
+## 2. Scope of work
+
+The MiniApp Components specification aims to collect the minimum set of common features from all the MiniApp versions, including: 
+
+- Built-in components (elements), names, and properties;
+- Event handling and basic event types;
+- High-level features for component definition;
+- Lifecycle.
+
+### Out of scope
+
+- Definition of HTML elements
+- Definition of behavior or appearance of elements.
+- Definition of a markup language for MiniApp
+
+### What kind of document will be produced?
+
+To avoid overlap and confusion with Web Components and other recommendations in W3C, this specification aims to be an informative reference and is not intended to be a formal standard. Thus, this specification will be published as a Group Note with a formal review but not be endorsed by W3C.
+
+This document will follow the principles below:
+
+- Identify the common practices and existing implementations regarding MiniApp components;
+- Maximize reuse and alignment of the specification with the existing standards (especially HTML, CSS, and DOM); and
+- Alignment with other groups in W3C (i.e., [OpenUI CG](https://open-ui.org/)) providing feedback and collecting new requirements that could serve to enhance the existing standards.
+
+
+## 3. Key considerations
+
+### Differences with Web Components
+
+This section analyzes the differences between the standard Web Components and the MiniApp components, considering the existing vendor-specific implementations, Baidu Smart Mini Programs, Alipay Mini Programs, and Quick Apps (Huawei, Xiaomi). 
+
+The main differences identified are:
+
+- __Component resources__. MiniApps require specific resource types, including markup languages and file system structure. 
+- __Component interface__. MiniApp components have specific properties and methods. Access to global variables and page elements differs in all versions. 
```

Comment:
```suggestion
- __Component interface__. MiniApps components have specific properties and methods. Access to global variables and page elements differs in all versions. 
```

Reply:
You keep using MiniApp and MiniApps. I would agree on one (plural), since you're referring to MiniApps as the concept, not as one miniapp or many miniapps.